From f17f772c4d6c86bfe240b5080decda97ffd80522 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Thu, 19 Sep 2024 17:35:01 +0200
Subject: [PATCH 1246/1695] dt-bindings: clock: stm32mp25: add IDs of
 Cortex-CM0 C3 clocks

Add CPU3 clock binding entry for STM32MP25 SoC family.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 734049719f1a996752d6e286303f6a49f742534a dt-bindings: clock: stm32mp25: add IDs of Cortex-CM0 C3 clocks]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: Iccff78dc2be3a9b6f6e4d1f65e2a4cfc0ab7ecc1
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/406045
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 .../dt-bindings/clock/stm32mp25-clks.h        | 38 ++++++++++++++++++-
 1 file changed, 37 insertions(+), 1 deletion(-)

diff --git a/core/include/dt-bindings/clock/stm32mp25-clks.h b/core/include/dt-bindings/clock/stm32mp25-clks.h
index a8d28b325..2cdc1df3e 100644
--- a/core/include/dt-bindings/clock/stm32mp25-clks.h
+++ b/core/include/dt-bindings/clock/stm32mp25-clks.h
@@ -365,8 +365,37 @@
 #define HSE_DIV2_CK		343
 #define CK_KER_ETR		344
 #define CK_KER_STM		345
+#define HSI_KER_CK		346
+#define HSE_KER_CK		347
+#define MSI_KER_CK		348
+#define CK_CPU3			349
+#define CK_ADF1_C3		350
+#define CK_GPIOZ_C3		351
+#define CK_I2C8_C3		352
+#define CK_I3C4_C3		353
+#define CK_LPDMA_C3		354
+#define CK_LPTIM3_C3		355
+#define CK_LPTIM4_C3		356
+#define CK_LPTIM5_C3		357
+#define CK_LPUART1_C3		358
+#define CK_RTC_C3		359
+#define CK_SPI8_C3		360
+#define CK_ADF1_AM		361
+#define CK_CPU3_AM		362
+#define CK_GPIOZ_AM		363
+#define CK_I2C8_AM		364
+#define CK_I3C4_AM		365
+#define CK_IPCC2_AM		366
+#define CK_LPDMA_AM		367
+#define CK_LPTIM3_AM		368
+#define CK_LPTIM4_AM		369
+#define CK_LPTIM5_AM		370
+#define CK_LPUART1_AM		371
+#define CK_RTC_AM		372
+#define CK_SPI8_AM		373
+
+#define STM32MP25_LAST_CLK	374
 
-#define STM32MP25_LAST_CLK	346
 
 #define CK_SCMI_ICN_HS_MCU	0
 #define CK_SCMI_ICN_SDMMC	1
@@ -492,5 +521,12 @@
 #define CK_SCMI_BUS_STM		122
 #define CK_SCMI_KER_STM		123
 #define CK_SCMI_KER_ETR		124
+#define CK_SCMI_HSI_KER_CK	125
+#define CK_SCMI_HSE_KER_CK	126
+#define CK_SCMI_MSI_KER_CK	127
+#define CK_SCMI_GPIOZ_AM	128
+#define CK_SCMI_IPCC2_AM	129
+#define CK_SCMI_LPDMA_AM	130
+#define CK_SCMI_RTC_AM		131
 
 #endif /* _DT_BINDINGS_STM32MP25_CLKS_H_ */
-- 
2.47.2

